pkg/common/resources/resources.go

```diff
@@ -455,6 +455,74 @@ func (r *Resource) fitIn(smaller *Resource, skipUndef bool) bool {
 	return true
 }
 
+// Denominator Resources can be either guaranteed Resources or fairmax Resources.
+// If the quanity is explicitly 0 or negative, we will check usage.  If usage >= 0, the share will be set to 1.0.  Otherwise, it will be set 0.0.
+func getShareFairForDenominator(resourceType string, allocated Quantity, denominatorResources *Resource) (float64, bool) {
+	if denominatorResources == nil {
+		return 0.0, false
+	}
+
+	denominator, ok := denominatorResources.Resources[resourceType]
+
+	switch {
+	case ok && denominator <= 0:
+		if denominator < 0 {
```

Comment:
Can we remove the logging? This is hot-path code, even evaluating whether log level is active is pretty expensive. Also, if you have n queues to compare you're going to emit ~ n log(n) debug calls per sort() invocation. Take that times every scheduling cycle, and you're going to emit millions of log entries very quickly if someone is unfortunate enough to turn this on.

Reply:
+1, we need no logging here, even if it's disabled

Reply:
done